sup3r/qa/bias_stats.py

```diff
@@ -0,0 +1,246 @@
+                check = (out_path is not None and not self.overwrite
+                         and os.path.exists(out_path))
+                if check:
+                    logger.info(f'Loading {out_path}.')
+                    dfs[dset][stat] = pd.read_csv(out_path)
+
+                else:
+                    logger.info(f'Computing {stat} for {dset} for {files}.')
+                    dfs[dset][stat] = TemporalStats.run(
+                        files, dset, statistics=stat, diurnal=False,
+                        month=self.do_months, combinations=False,
+                        res_cls=self.RESOURCE_CLASS,
+                        max_workers=self.max_workers, chunks_per_worker=5,
+                        lat_lon_only=True, mask_zeros=False, out_path=out_path)
+        return dfs
+
+    def compute_stats(self):
+        """Compute stats for sup3r and reference files.
+
+        Returns
+        -------
+        sup3r_dfs : dict
+            Dictionary of pandas dataframes with keys for each dataset and
+            stat for sup3r files. e.g. sup3r_dfs['windspeed_10m']['mean'].
+        ref_dfs : dict
+            Dictionary of pandas dataframes with keys for each dataset and
+            stat for reference files. e.g. ref_dfs['windspeed_10m']['mean'].
+        """
+        logger.info(f'Computing stats for sup3r_file: {self.sup3r_file}.')
+        sup3r_dfs = self._compute_stats(self.sup3r_file, res_name='sup3r')
+        logger.info(f'Computing stats for ref_file: {self.ref_file}.')
+        ref_dfs = self._compute_stats(self.ref_file, res_name='ref')
+        return sup3r_dfs, ref_dfs
+
+    def compute_biases(self, sup3r_dfs, ref_dfs):
+        """Compute biases between sup3r and reference stats
+
+        Parameters
+        ----------
+        sup3r_dfs : dict
+            Dictionary of pandas dataframes with keys for each dataset and
+            stat for sup3r files. e.g. sup3r_dfs['windspeed_10m']['mean'].
+        ref_dfs : dict
+            Dictionary of pandas dataframes with keys for each dataset and
+            stat for reference files. e.g. ref_dfs['windspeed_10m']['mean'].
+
+        Returns
+        -------
+        bias_dfs : dict
+            Dictionary of pandas dataframes with bias values with keys for each
+            dataset and stat. e.g. sup3r_dfs['windspeed_10m']['mean'][col] -
+            ref_dfs['windspeed_10m']['mean'][col]
+        """
+        bias_dfs = {}
+        for dset in sup3r_dfs:
+            bias_dfs[dset] = {}
+            for stat, df in sup3r_dfs[dset].items():
+                sup3r_df = df.copy()
+                ref_df = ref_dfs[dset][stat]
+                cols = sup3r_df.columns
+                cols = set(cols).intersection(ref_df.columns)
+                cols = [col for col in cols if col
+                        not in ('latitude', 'longitude')]
+                bias_df = sup3r_df.iloc[self.sup3r_sites]
+                bias_df[cols] -= ref_df.iloc[self.ref_sites][cols].values
```

Comment:
Is this just delta bias then? How is this different than the code in `sup3r/bias/bias_calc.py`? Do we really need another class for this? My understanding is that this just does a KDTree and diff between `TemporalStats` outputs?